R/FilterStateChoices.R

```diff
@@ -552,12 +557,12 @@ ChoicesFilterState <- R6::R6Class( # nolint
                 if(private$multiple) {
                   updateCheckboxGroupInput(
                     inputId = "selection",
-                    selected = private$selected()
+                    selected = private$get_selected()
                   )
                 } else {
                   updateRadioButtons(
                     inputId = "selection",
-                    selected =  private$selected()
+                    selected =  private$get_selected()[1]
```

Comment:
Is the index necessary? `radioButtons` always returns one value, therefore so will `$get_selected`.

Reply:
Why would `$get_selected` return more than one value if `multiple = FALSE`?